### PR TITLE
[15.0][MIG] survey_crm_generation: Migration to version 15.0

### PR DESCRIPTION
**Pull request by stefan-tecnativa**
cc @Tecnativa TT43681

@chienandalu please review!

### CODE REVIEW

File: survey_crm_generation/tests/test_survey_crm_sale_generation.py [hunk 1]
```diff
@@ -37,8 +37,10 @@ def test_lead_generation(self):
...
         )
         expected_lead_description = (
-            "E-mail address: test@test.com\n"
+            "<p>E-mail address: test@test.com\n"
             "Your company name?: Tecnativa\n"
-            "And your name?: Tecnativa"
+            "And your name?: Tecnativa</p>"
+        )
+        self.assertEqual(
+            str(self.generated_lead.description), expected_lead_description
```
Comment (chienandalu) [inline survey_crm_generation/tests/test_survey_crm_sale_generation.py]: Better convert to a Markup object the expected_lead description

**Review by chienandalu — APPROVED** [review]

File: survey_crm_generation/tests/test_survey_crm_sale_generation.py (hunk 1 above)
Comment (stefan-tecnativa) [reply survey_crm_generation/tests/test_survey_crm_sale_generation.py]: Done, thanks for the suggestion

File: survey_crm_generation/tests/test_survey_crm_sale_generation.py [hunk 2]
```diff
@@ -1,5 +1,7 @@
 # Copyright 2023 Tecnativa - David Vidal
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+import markupsafe
```
Comment (chienandalu) [inline survey_crm_generation/tests/test_survey_crm_sale_generation.py]:
```suggestion
from markupsafe import Markup
```

File: survey_crm_generation/tests/test_survey_crm_sale_generation.py [hunk 3]
```diff
@@ -37,8 +39,11 @@ def test_lead_generation(self):
             ),
         )
         expected_lead_description = (
-            "E-mail address: test@test.com\n"
+            "<p>E-mail address: test@test.com\n"
```
Comment (chienandalu) [inline survey_crm_generation/tests/test_survey_crm_sale_generation.py]:
```suggestion
        expected_lead_description = Markup(
            "<p>E-mail address: test@test.com\n"

Comment (stefan-tecnativa) [reply survey_crm_generation/tests/test_survey_crm_sale_generation.py]: You're right, it's cleaner this way

**Review by chienandalu — APPROVED** [review]
Thanks

@victoralmau can you review?

**Review by victoralmau — APPROVED** [review]

**Pull request merged**